types/promise.prototype.finally/index.d.ts

```diff
@@ -1,11 +1,11 @@
-// Type definitions for promise.prototype.finally v2.0.1
+// Type definitions for promise.prototype.finally 2.0
```

Comment:
This change avoids lint error. Lint rules does not permit includes patch version and "v" letter.

```
ERROR: 1:25   dt-header   Error parsing header. Expected: foo MAJOR.MINOR (patch version not allowed)
ERROR: 1:25   dt-header   Error parsing header. Expected: foo MAJOR.MINOR ('v' not allowed)
```